frontend/src/views/Crops/CropsModal.tsx

```diff
@@ -469,6 +485,69 @@ function CropsModal({
     return [{ id: Math.random(), reqN: remN, reqP2o5: remP2o5, reqK2o: remK2o }];
   }, [formData]);
 
+  // Check if N can be edited (Field vegetables with modifynitrogen = true AND calculations have been performed)
+  const isNEditable = selectedCropType?.modifynitrogen === true && calculationsPerformed;
+
+  // Dynamic columns for requirement table - make N editable for Field vegetables
+  const requirementColumns: GridColDef[] = useMemo(
+    () => [
+      {
+        field: 'reqN',
+        headerName: 'N',
+        width: 75,
+        minWidth: 75,
+        maxWidth: 200,
+        sortable: false,
+        resizable: false,
+        flex: 1,
+        ...(isNEditable && {
+          renderCell: (params) => (
+            <NumberField
```

Comment:
I don't entirely know where reqN is set, but everywhere else that the prop is set other than here (I assume in the handleCalculate method, maybe elsewhere) you need to set reqNAdjusted to false.

For example, a person could calculate once, change this field, mess around with the yield, then hit calculate again. The N value would be reset to the default on the second calculate.